core/frontend/src/main/scala/io/udash/bindings/modifiers/SeqPropertyModifier.scala

```diff
@@ -1,16 +1,44 @@
 package io.udash.bindings.modifiers
 
-import io.udash.bindings._
+import com.avsystem.commons.SharedExtensions._
+import io.udash.bindings.Bindings._
 import io.udash.properties._
 import io.udash.properties.seq.{Patch, ReadableSeqProperty}
 import io.udash.properties.single.ReadableProperty
-import org.scalajs.dom
 import org.scalajs.dom._
 
-import scalatags.generic._
+import scala.scalajs.js
 
-private[bindings] class SeqPropertyModifier[T, E <: ReadableProperty[T]]
-  (property: ReadableSeqProperty[T, E], builder: E => Seq[Element]) extends Modifier[dom.Element] with Bindings {
+private[bindings]
+class SeqPropertyModifier[T, E <: ReadableProperty[T]](property: ReadableSeqProperty[T, E],
+                                                       builder: (E, Binding => Binding) => Seq[Element])
+  extends Binding {
+
+  def this(property: ReadableSeqProperty[T, E], builder: E => Seq[Element]) = {
+    this(property, (d, _) => builder(d))
+  }
+
+  protected val nestedBindingsByProperty: js.Dictionary[js.Array[Binding]] = js.Dictionary.empty
```

Comment:
Any reason to use a `.js.Dictionary` (and earlier an `js.Array`) instead of Scala collections?

Reply:
It's probably premature optimization, but native collections are faster.